docs/source/use_cases/privacy_protection_people_screens.rst

```diff
@@ -0,0 +1,128 @@
+**********************************************
+Privacy Protection (People & Computer Screens)
+**********************************************
+
+Overview
+========
+
+Videos and Pictures often contain people and other sensitive visual information (e.g., the display
+on computer screens), even though this information might not be needed at all for visual
+processing. Our solution performs full body anonymization and computer screen blurring to protect
+the identities of individuals, and the sensitive information on computer screens from footage or
+pictures. It can be used to comply with the General Data Protection Regulation (GDPR) or other data
+privacy laws.
+
+.. image:: /assets/use_cases/privacy_protection_people_screens.gif
+   :class: no-scaled-link
+   :width: 50 %
+
+In this example use case, we want to count the number of people in the office, but we don't want
+compromise the privacy of the office inhabitants, nor the privacy of any information accidentally
+shown on the computer screens.
```

Comment:
```suggestion
In this example use case, we want to count the number of people in the office, but also want to avoid
compromising the privacy of the office inhabitants or information displayed on computer screens.
```